src/test/java/com/guysfromusa/carsgame/v1/CarResourceTest.java

```diff
@@ -102,9 +102,8 @@ public void shouldAddCarToGame(){
         ResponseEntity<Car> carResponseEntity = template.exchange(url, POST, requestEntity, Car.class);
 
         //then
-        Car responseEntityBody = carResponseEntity.getBody();
-        assertThat(responseEntityBody.getPosition())
-                .extracting(Point::getX, Point::getY).contains(1, 1);
+        Car carResponse = carResponseEntity.getBody();
+        assertThat(carResponse.getPosition()).extracting(Point::getX, Point::getY).contains(1, 1);
```

Comment:
try to keep format assertThat(carResponse.getPosition())
.extracting(Point::getX, Point::getY)
.contains(1, 1);
for better readability